github_connector/models/github.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# @author: Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import exceptions, _
+
+from requests.auth import HTTPBasicAuth
+import requests
+import json
+import logging
+
+_logger = logging.getLogger(__name__)
+
+_MAX_NUMBER_REQUEST = 30
+
+_BASE_URL = 'https://api.github.com/'
+
+_GITHUB_URL = 'https://github.com/'
+
+_GITHUB_TYPE = [
+    ('organization', 'Organization'),
```

Comment:
could you be more precise ? If you're talking about PR labels, it's not necessary at this point, because PR and issues are not synchronized for the time being.